apps/api/v2/src/modules/teams/memberships/inputs/update-team-membership.input.ts

```diff
@@ -1,17 +1,20 @@
-import { ApiProperty } from "@nestjs/swagger";
+import { ApiProperty, ApiPropertyOptional } from "@nestjs/swagger";
 import { MembershipRole } from "@prisma/client";
 import { IsBoolean, IsOptional, IsEnum } from "class-validator";
 
 export class UpdateTeamMembershipInput {
   @IsOptional()
   @IsBoolean()
+  @ApiPropertyOptional({ type: Boolean, default: false })
   readonly accepted?: boolean = false;
 
+  @IsOptional()
   @IsEnum(MembershipRole)
-  @ApiProperty({ enum: ["MEMBER", "OWNER", "ADMIN"] })
+  @ApiPropertyOptional({ enum: ["MEMBER", "OWNER", "ADMIN"], default: MembershipRole.MEMBER })
   readonly role?: MembershipRole = MembershipRole.MEMBER;
```

Comment:
❗I think since its an update we can remove setting default value and have on line 14
```
readonly role?: MembershipRole;
```